lib/utils/constants.dart

```diff
@@ -15,6 +26,43 @@ class AppStrings {
   static const String PRESENTATION_TEXT4 = 'E muito mais! Vamos descobrir?';
 }
 
+class ErrorMessage {
```

Comment:
Essas mensagens são adicionadas no log ou são mostradas ao usuário? Se for o segundo caso, o ideal é que sejam mostradas em português.

Reply:
Resolvido